.travis.yml

```diff
@@ -7,3 +7,4 @@ rvm:
   - 2.1.0
   - 2.2.0
   - 2.3.0
+before_script: bundle exec rake build && [[ -z $(git status -s) ]]
```

Comment:
@tullis This is relying on the fact that `git status -s` outputs nothing if everything is ok. But if it _does_ output something, the output is lost. Is there a nifty way we can both keep the `-z` test but also see the output?

If anyone is reading this and has a better suggestion for how to accomplish this, please pipe up!